developer_copr_setup.md

```diff
@@ -0,0 +1,157 @@
+## Developer Copr Setup:
+
+### Background
+
+ManageIQ requires some upstream rpms for use on our CentOS appliances.
+Copr makes these rpms publicly available for the community build.
+
+These steps will walk you through:
+
+* Create a fedora / copr account
+* Create a copr project
+* add repositories for dependencies
+* specify the chroot
+* add any additional packages to the minimal chroot
+* specify the `src.rpm` to build
+* host existing source rpm (srpm) publicly
+* mock, test, and build the rpm
+* incorporate the copr yum repo into our upstream community build.
+
+
+###  Create a fedora account (used as a copr account)
+* Create a fedora account: http://copr.fedoraproject.org/
```

Comment:
The grammar bug in me is dying from the lack of capitalization of proper nouns :D